src/webapp-lib/app.pug

```diff
@@ -14,9 +14,11 @@ html(lang="en")
         img(src=require('!url-loader?mimetype=image/svg+xml!cocalc-icon-white-transparent.svg'))
       div.banner-error
         div.message
-          | Something bad must have happened.
+          | Timeout while loading CoCalc.
```

Comment:
One funny thing about this "timeout" is that if your network is slow and it takes 65 seconds to load the page, then it'll show this error, wait 5s and still load fine.  So it's only a timeout in the sense that it's "suspicious".  I've seen this a lot when doing dev (via cell phone network), where the page load size went to 15MB+...  That's why it's 60s and not 30s.